projects/arc/src/app/main/main.component.html

```diff
@@ -8,7 +8,11 @@
             class="cursor-pointer menu-icon"
             (click)="toggle()"
           ></nb-icon>
-          <a class="logo" href="javascript:void(0)" (click)="navigateHome()">
+          <a
+            class="logo"
+            href="javascript:void(0)"
+            (click)="navigateHome('/main/home')"
```

Comment:
hard coded values should not be used

Reply:
i did'nt change any code i change path only .still will see